src/lando/utils/management/commands/setup_dev.py

```diff
@@ -19,6 +19,7 @@ def setup_workers(self):
             SCM_TYPE_GIT: {
                 WorkerType.LANDING: "",
                 WorkerType.AUTOMATION: "-automation-worker",
+                WorkerType.UPLIFT: "-uplift-worker",
```

Comment:
What's the plan for Thunderbird uplifts? Phabricator way until migration? I don't think old-style uplifts are/will be requestable from Lando anymore?